src/App.jsx

```diff
@@ -14,70 +17,84 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
```

Comment:
Replace with `goods[goods.indexOf('Jam')] || goods[0]`

It's more a safe variant.